cmd/sidecars/kafka/client/writer_test.go

```diff
@@ -92,6 +92,7 @@ func TestWriter_WriteMessage(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			writer := &Writer{
 				producer: tt.fields.producer,
+				metrics:  map[string]writerMetrics{},
```

Comment:
map[...],,{} -> make(map[...]...)

Reply:
I think this leads to the same thing, no?

Reply:
I changed it, anyway